packages/experiments/src/utilities/BaseState.tsx

```diff
@@ -0,0 +1,85 @@
+import { IStateComponentProps } from '../Foundation';
+import { BaseComponent } from '../Utilities';
```

Comment:
It would be useful to have a markdown file explaining examples of creating components both stateless and stateful.